### ISSUE TITLE
mobile: "Shared with me" shows "All" albums

### ISSUE THREAD

**Issue by C-Otto** [immich-app/immich#13466]
### The bug

I navigated to "Albums" - "Shared with me". I saw the correct list of albums for a second or so, then the list was replaced so that I saw all albums (not just the ones shared with me). I assume that the "all albums" information was requested in the background and, once completed, this populated the view, ignoring the navigation that happened in between.

### The OS that Immich Server is running on

Debian Stable

### Version of Immich Server

v1.118.0

### Version of Immich Mobile App

1.118.0 build.163 (Android)

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
x
```

### Your .env content

```Shell
x
```

### Reproduction steps

1. Have many albums
2. Navigate to "Albums" - "Shared with me"

**Comment by alextran1502** [comment]
Yeah I just noticed that. Looks like when the album sync is in progress it would update the state of the filter

**Comment by yashrajjain726** [comment]
Yes, this is happening during the album sync only, not always.

**Comment by thomaswucher** [comment]
Can confirm this bug still exists on app version 1.135.1.

**Comment by jrasm91** [comment]
This has been fixed in the new timeline implementation.
